programs/models.py

```diff
@@ -2,21 +2,51 @@
 from phonenumber_field.modelfields import PhoneNumberField
 from translations.models import Translation
 from programs.programs import calculators
+from programs.programs.fpl import FplCache
 from programs.util import Dependencies, DependencyError
 
+import requests
+from integrations.util.cache import Cache
+
+
+class FplCache(Cache):
+    expire_time = 60 * 60 * 24  # 24 hours
+    default = {}
+    api_url = "https://aspe.hhs.gov/topics/poverty-economic-mobility/poverty-guidelines/api/"
+    max_household_size = 8
+
+    # need to obtain the FPL for each of these years for hhSizes 1-8 and return that as a dictionary to match the current fpl dictionary
+    def update(self):
+        # need to obtain all of the years that we have data for on the backend
+        # we do this by querying the backend this needs to happen in update because anything below these functions won't run
```

Comment:
`'''` comments are used to document what a function/method does. The description will appear when you hover over the function/method.
```suggestion
    def update(self):
        '''
        Get FPLs for all relevant years using the official FPL API
        '''
        # retrieve all of the FPLs that have a program attached.
```